### DISCUSSION TITLE
Automatic translation of documentation

### DISCUSSION THREAD

**Discussion by technokratos** [hneemann/Digital#724] [General]
Where can I find the source code for the documentation in a more readable format than pdf?
I'm working on Russian translation of the program.
The resource translation did not cause a problem, but PDF format of documentation is not convenient for automatic translation.

**Comment by hneemann** [comment]
see [here](https://github.com/hneemann/Digital/tree/master/src/test/resources/docu)

**Reply by hneemann** [reply]
I don't think that will work well. Automatic translators can't cope well with the required technical terms.

**Reply by technokratos** [reply]
Thanks. 
Of course, automatic translation isn't the best, but I can start using the result in one day and can fix remarks in the future on the feedbacks from my students.

**Reply by hneemann** [reply]
Please keep in mind to keep the original English files as reference files "_ref.xml".
This will allow me to accept a pull request in the future. Without the reference files it is impossible to keep track of further changes.

**Reply by technokratos** [reply]
I made pull request with translation GUI and documentation.
I small changed FOP generation, without directly declare Arial font it's not work with Cyrillic.
I'm ready to support Russian translation in the future.
